lib/tests/streamlit/elements/doc_string_test.py

```diff
@@ -48,3 +62,21 @@ def test_st_help(self):
             el.doc_string.startswith("Change the current working directory")
         )
         self.assertEqual(f"posix.chdir(path)", el.value)
+
+    def test_st_help_with_available_conditional_members(self):
+        """Test st.help with conditional members available"""
+
+        st.help(ConditionalHello(True))
+        el = self.get_delta_from_queue().new_element.doc_string
+        self.assertEqual("ConditionalHello", el.type)
+        member_names = [member.name for member in el.members]
+        self.assertTrue("say_hello" in member_names)
```

Comment:
nit (here and below): I think there's an `assertIn` method that makes this more explicit